packages/labs/ssr/src/test/integration/tests/basic.ts

```diff
@@ -4698,6 +4698,178 @@ export const tests: {[name: string]: SSRTest} = {
     };
   },
 
+  'LitElement: Reflected number attribute': () => {
+    return {
+      registerElements() {
+        class LEReflectedNumberAttribute extends LitElement {
+          @property({type: Number, reflect: true})
+          num = 42;
+        }
+        customElements.define(
```

Comment:
why not use the decorator?

Reply:
Because none of the existing test cases here do